app/calculate_chart.py

```diff
@@ -17,22 +17,28 @@
 #plt.rc('text', usetex=True)
 plt.rc('font', family='sanserif')
 
-def plot_psychro(temp_air = np.arange(10, 35, .2), 
-				RH_psy = np.arange(0,100,.2)/100, 
+def plot_psychro(temp_air_init = np.arange(10, 35, .5), 
```

Comment:
Could you add a littie comment below the function defining what all the inputs are. For example

```python
'''
Input
MR: Metabolic Rate [J/s or whatever the unit is]
w: Skin wettedness [unit]
 ...
'''
```

If you want a really detailed example, have a look at this function https://github.com/architecture-building-systems/RC_BuildingSimulator/blob/923ff3d83bb30684c212b36afccfe70c89a1022e/rc_simulator/radiation.py#L45

Reply:
added